terminus/geometry/arc.py

```diff
@@ -70,12 +85,45 @@ def center_point(self):
                        self._radius * direction_multiplier * math.cos(theta_in_radians))
         return self._start_point + vector
 
+    def circle(self):
+        return Circle(self.center_point(), self.radius())
+
     def length(self):
         """
         The length of the perimeter of the arc
         """
         return abs(math.pi * self._radius * self._angular_length / 180.0)
 
+    def _clip_angle_to_360(self, angle):
+        return angle % 360
+
+    def counter_clockwise(self):
+        """
+        If the arc has clockwise direction, it returns the same arc traveled in
+        counter-clockwise direction. If it already has counter-clockwise direction,
+        it returns the same arc.
+        """
+        if self.angular_length() >= 0:
+            # Nothing to do, already counter clockwise
+            return self
+        return Arc(self.end_point(), self.end_heading() + 180, self.radius(), -self.angular_length())
+
+    def contains_arc(self, other):
+        # this method doesn't consider arc directions
+        if not self.circle().almost_equal_to(other.circle()):
+            return False
+        elif (not self.includes_point(other.start_point())) or (not self.includes_point(other.end_point())):
+            return False
+        else:
+            arc1 = self.counter_clockwise()
+            arc2 = other.counter_clockwise()
+            if arc1.almost_equal_to(arc2):
+                return True
+            elif arc2.includes_point(arc1.start_point()) and arc2.includes_point(arc1.end_point()):
+                return False
+            else:
+                return True
+
```

Comment:
Note that you are returning a boolean based on a boolean check. This type of things can be rewritten using boolean logic to avoid unnecessary ifs. Something along the lines of:

``` 
return arc1.almost_equal_to(arc2) or (....)
```